time_tracker.py

```diff
@@ -3,11 +3,9 @@
 import time
 import os
 
-
```

Comment:
according to the python style guide (https://www.python.org/dev/peps/pep-0008/), you should have two empty lines between functions.